src/components/Article.js

```diff
@@ -81,16 +81,16 @@ const ArticleInner = ({ lang, articleTitle, initialSubTitle }) => {
     return 'Loading...'
   }
 
-  const [isTocShown, toggleToc] = useState(false)
   const [subTitle, setSubTitle] = useState(initialSubTitle)
+  const [popupToc] = usePopup(ArticleToc)
```

Comment:
For consistency, the function here should be named show<Something>Popup(). For instance `showArticleTocPopup` or just `showTocPopup`

Reply:
see if it is better now